test/JDBC/input/functions/typeproperty.sql

```diff
@@ -0,0 +1,206 @@
+--Cleanup
```

Comment:
we can remove this redundant test file since we already have tests in vu-prepare/verify/cleanup format.

Reply:
Removed